src/commands.cpp

```diff
@@ -19,6 +19,7 @@
  * program.
  */
 
+#include "megaapi.h"
```

Comment:
as megaapi.h is the public interface, we can't have a dependency on it in the core code

Reply:
Will fix